api/bucketsd/service.go

```diff
@@ -3343,6 +3345,7 @@ func toPbArchiveConfig(config *mdb.ArchiveConfig) *pb.ArchiveConfig {
 			MaxPrice:        config.MaxPrice,
 			FastRetrieval:   config.FastRetrieval,
 			DealStartOffset: config.DealStartOffset,
+			VerifiedDeal:    config.VerifiedDeal,
```

Comment:
Missing mapping mentioned in the PR description!

Reply:
👍